src/modelfree/configs/multi/score.py

```diff
@@ -165,6 +175,59 @@ def mask_observations_with_zeros(exp_name, score):
         score['mask_agent_kwargs'] = {'masking_type': 'zeros'}
         exp_name = 'zero_' + exp_name
 
+    @multi_score_ex.named_config
+    def mask_observations_with_additive_noise(exp_name, score, spec):
+        score = score.copy()
```

Comment:
Why are we using `score = score.copy()` here rather than `score = dict(score)`?

I don't particularly care which we use, but we should be consistent throughout. If there's a good reason for using `score.copy()` lets change all of them to that; otherwise keep it as `dict(score)`?

Reply:
Okay